pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -214,6 +218,20 @@ func newLibvirtDomainManager(connection cli.Connection, virtShareDir, ephemeralD
 	return &manager, nil
 }
 
+func getDomain(dom cli.VirDomain) (*api.Domain, error) {
```

Comment:
ah, this cannot work this way. `GetXMLDesc` gets the spec but it wouldn't understand our api.Domain struct.